src/containers/PharmacyList/PharmacyList.test.tsx

```diff
@@ -46,10 +46,10 @@ describe("PharmacyList", () => {
     const phones = await screen.findAllByText(/\(613\) 555-5555/i);
     expect(phones.length).toBe(4);
 
-    const links = await screen.findAllByRole("link", {
-      name: /https:\/\/idapharmacy\.com\//i,
-    });
-    expect(links.length).toBe(4);
+    // const links = await screen.findAllByRole("link", {
```

Comment:
I just pulled this down to see if I could fix the query, but given the markup generated, there appears to be no query that can target the new link buttons... I tried fiddling with some attributes, but couldn't get anything working, there are too many `span`s for RTL to be able to target anything.

We could cut a ticket to make a RTL-compatible button, but for the time being we can delete this dead code and move ahead.